### PR TITLE
Add option to not list article on widgets

### PR DESCRIPTION
**Pull request by JoaRiski**
Add the possibility to not list an article on the article widgets
This is done by adding a boolean field/checkbox to the article model

Refs [SER-3446](https://project.anders.fi/browse/SER-3446)

### CODE REVIEW

**Review by ehaivala — COMMENTED** [review]
Looks simple enough. Unit test case would be nice though. :)

**Comment by JoaRiski** [comment]
Test added

File: tests/test_plugins.py [hunk 1]
```diff
@@ -47,6 +47,23 @@ def test_article_list_plugin_article_count():
 
 
 @pytest.mark.django_db
+def test_article_list_plugin_show_flag():
+    """
+    Test article list plugin showing only the articles with the
+    show_in_article_list_plugin flag set to True
+    """
+    create_articles(3)
+    hidden_article = Article.objects.get(pk=3)
```
Comment (ehaivala) [inline tests/test_plugins.py:56]: Use `last()` instead of `get(pk=3)`

**Comment by JsseL** [comment]
Nice code and good stuff, but won't accept :smile: As talked IRL, consider adding the exclude filters to the generic list latest plugin.

File: tests/test_plugins.py (hunk 1 above)
Comment (JoaRiski) [reply tests/test_plugins.py:56]: thought so, but IIRC `last()` was quite new feature so chose to use more backward-compatible instead. This PR is probably obsolete now anyway but thanks for the review.

**Comment by JoaRiski** [comment]
Closing in favour of #13

**Pull request closed without merging**